apps/docs/src/app/core/component-docs/content-density-2/content-density-user/content-density-user.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Inject, ViewEncapsulation } from '@angular/core';
```

Comment:
maybe we should remove to `content-density-example-consumer`?

Reply:
Nah, it works too 😁